packages/compose/src/utils/debounce/index.ts

```diff
@@ -0,0 +1,220 @@
+export interface DebounceOptions {
```

Comment:
Let's include the license from `lodash` at the top of the file similar to what we did in https://github.com/WordPress/gutenberg/pull/44112.

Reply:
Good point, added in https://github.com/WordPress/gutenberg/pull/43943/commits/80826baaafd43cac8a71b0e47768559eede9d1a3